tests/test_simplelog.py

```diff
@@ -0,0 +1,192 @@
+import sys
+
+import re
+import click
+import logging
+
+log_stream = None
+
+
+def tool_resetlog():
+
+    global log_stream
```

Comment:
That's a good thing, but you can freely disobey PEP8 if you feel your code is better off without it. Also I'd recommend using the ``pep8`` package instead of checking manually.